web-client/src/ustc-ui/Tabs/Tabs.jsx

```diff
@@ -63,11 +63,13 @@ export function TabsComponent({
 
   const renderTab = child => {
     const {
+      className,
```

Comment:
'className' is already declared in the upper scope on line 39 column 3

Reply:
Good catch. Reassigned on destructure so it isn't shadowing the var in the upper scope.